src/electron/views/automated-checks/components/title-bar.tsx

```diff
@@ -0,0 +1,57 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import { BrowserWindow } from 'electron';
+import { ActionButton } from 'office-ui-fabric-react/lib/Button';
+import * as React from 'react';
+
+import { NamedFC } from 'common/react/named-fc';
+import { BrandWhite } from 'icons/brand/white/brand-white';
+import { titleBar } from './title-bar.scss';
+
+export type TitleBarDeps = {
+    currentWindow: BrowserWindow;
+};
+
+export interface TitleBarProps {
+    deps: TitleBarDeps;
+}
+
+export const TitleBar = NamedFC<TitleBarProps>('TitleBar', (props: TitleBarProps) => {
+    const currrentWindow = props.deps.currentWindow;
+    const minimize = () => currrentWindow.minimize();
+    const maximize = () => (currrentWindow.isMaximized() ? currrentWindow.restore() : currrentWindow.maximize());
+    const close = () => currrentWindow.close();
+
+    return (
+        <div className={titleBar}>
+            <BrandWhite />
+            <ActionButton
```

Comment:
Looking at the `ActionButton` properties I noticed `ariaHidden={true}` and `tabIndex={-1}`.
I think that's correct, Fernanda & me check edge and teams and this looks like the expected behavior) but I'm worried about keyboard shortcuts for this actions. I'm guessing closing the windows should just work; alt + F4 looks universal enough (on Windows) that I'm not too worried about but I'm wondering about how to minimize and maximize.

Reply:
At least on windows, we've got win + down/up arrow which transition between max/restore/min as well as alt + space which gives all the options in a menu.